project/chrome-extension/script-two.js

```diff
@@ -18,4 +20,17 @@ function wordFoundAndIntro(){
    const intro = document.getElementById("wordTitle");
     //Adds the word of the id to the wordTitle element
    intro.innerText = "See the latest information about " + localStorage.getItem("rdoIdVarKey");
-}
\ No newline at end of file
+}
+
+// Add information aboout word to info.html
+function createDescription(){
+  const description = document.getElementbyId("description");
+  //description.innerText = ...;
```

Comment:
Do not leave commented out code when merging.

Reply:
We left these comments on purpose so that @SamRod33 and @SofiaPSU could know where the information from the JSON should go. After those were implemented they would be removed.

Reply:
In that case, this comment should be a todo:

`// TODO(username_for_who_has_to_fix_this): Add to the description by using description.innerText`

Also, do not forget that between comments and the first word, there must be a space.

Reply:
Noted, thank you!